libnavigation-ui/src/main/java/com/mapbox/navigation/ui/NavigationView.java

```diff
@@ -346,6 +357,14 @@ public void updateWayNameView(@NonNull String wayName) {
     wayNameView.updateWayNameText(wayName);
   }
 
+  /**
+   * Updates the text of the speed limit view
```

Comment:
Is this only updating the text of the speed limit view? What happens if any of the values from `SpeedLimit` is updated? Are we updating the whole view dynamically?

Reply:
This updates the entire speed limit view including any changes to background and text.

Reply:
Could we add a quick comment to the docs clarifying this behavior?